tests/Get-DbaBackupInformation.Tests.ps1

```diff
@@ -63,6 +63,7 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
     AfterAll {
         $null = Get-DbaDatabase -SqlInstance $script:instance1 -Database $dbname | Remove-DbaDatabase -Confirm:$false
```

Comment:
maybe here a single Get-DbaDatabase -database $dbname,$dbname2,$dbname3 is shorter and still effective